packages/react-ui-ag/src/Filters/__tests__/BathroomFilterCard-test.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import FilterCard from '../FilterCard'
+import theme from './mocks/theme'
+import ThemedBathroomFilterCard from '../BathroomFilterCard'
+
+const BathroomFilterCard = ThemedBathroomFilterCard.WrappedComponent
+
+describe('ag/Filters/BathroomFilterCard', () => {
+  it('has a default data-tid', () => {
+    const wrapper = shallow(<BathroomFilterCard />)
+    expect(wrapper.find('[data-tid="bathroom-filter-card"]')).toHaveLength(1)
+  })
+
+  it('allows data-tid to be overriden', () => {
+    const wrapper = shallow(
+      <BathroomFilterCard
+        data-tid="bathroom-filter-card-test"
+      />
+    )
+    expect(wrapper.find('[data-tid="bathroom-filter-card-test"]')).toHaveLength(1)
+    expect(wrapper.find('[data-tid="bathroom-filter-card"]')).toHaveLength(0)
+  })
+
+  it('renders radiogroups by default', () => {
```

Comment:
Not sure if we need this. This test is the same as the first.

Reply:
You're right - I was supposed to take that out too. Removing 👍